docs/references/subcommands/analyze.md

```diff
@@ -128,11 +128,12 @@ We support the following archive formats:
-| --------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
-| [`--detect-vendored`](./analyze/detect-vendored.md)                               | Enable the vendored source identification engine. For more information, see the [C and C++ overview](../strategies/languages/c-cpp/c-cpp.md).                            |
-| [`--detect-dynamic './some-binary`](./analyze/detect-dynamic.md)                  | Analyze the binary at the provided path for dynamically linked dependencies. For more information, see the [C and C++ overview](../strategies/languages/c-cpp/c-cpp.md). |
-| [`--static-only-analysis`](../strategies/README.md#static-and-dynamic-strategies) | Do not use third-party tools when analyzing projects.                                                                                                                    |
+| Name                                                                              | Description                                                                                                                                                                                                       |
+|-----------------------------------------------------------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| [`--detect-vendored`](./analyze/detect-vendored.md)                               | Enable the vendored source identification engine. For more information, see the [C and C++ overview](../strategies/languages/c-cpp/c-cpp.md).                                                                     |
+| [`--detect-dynamic './some-binary`](./analyze/detect-dynamic.md)                  | Analyze the binary at the provided path for dynamically linked dependencies. For more information, see the [C and C++ overview](../strategies/languages/c-cpp/c-cpp.md).                                          |
+| [`--static-only-analysis`](../strategies/README.md#static-and-dynamic-strategies) | Do not use third-party tools when analyzing projects.                                                                                                                                                             |
+| `--strict`                                                                        | Enforces strict analysis to ensure the most accurate results for a strategy by rejecting fallbacks. When ran with `--statitic-only-analysis`, the most optimal static strategy will be applied without fallbacks. |
```

Comment:
```suggestion
| `--strict`                                                                        | Enforces strict analysis to ensure the most accurate results by rejecting fallbacks. When run with `--static-only-analysis`, the most optimal static strategy will be applied without fallbacks. |
```

The removal of "accurate results for a strategy" is because you refer to a strategy as if it's a collection of different techniques in the first sentence, but then use it to refer to a component of that collection in the second. I don't feel too strongly about it, but if you think that might be confusing (or I'm just wrong), feel free to leave it as is.

Reply:
Could you also add a paragraph or two about this in the strategies [documentation](https://github.com/fossas/fossa-cli/blob/master/docs/references/strategies/README.md?plain=1#L131)? I think a section `### Strict Analysis` would be sufficient. A good thing to include would be an example of where the decision matters, such as Maven e.g. "For Maven, we will try an analysis using the available `mvn` tool, but if that fails FOSSA will try to statically analyze available pom files. With strict analysis, FOSSA will not attempt the static analysis and report failure if using `mvn` doesn't work."